server/src/main/java/org/elasticsearch/index/engine/EngineConfig.java

```diff
@@ -354,27 +350,6 @@ public LongSupplier getPrimaryTermSupplier() {
         return primaryTermSupplier;
     }
 
-    /**
-     * A supplier supplies tombstone documents which will be used in soft-update methods.
-     * The returned document consists only _uid, _seqno, _term and _version fields; other metadata fields are excluded.
-     */
```

Comment:
Maybe it's worth moving this javadoc to the `ParsedDocument` tombstone factory methods?

Reply:
+1. Adjusted in https://github.com/elastic/elasticsearch/pull/72593/commits/56cb22dd88f078fc188faf27a34dcab5e4884c37.